app/assets/javascripts/analytics/static-analytics.js

```diff
@@ -207,6 +209,26 @@
     this.setDimension(21, position);
   };
 
+  StaticAnalytics.prototype.setTotalNumberOfSections = function() {
+    var sidebarSections = $('.govuk-related-items h2').length;
```

Comment:
Small nit: having `a[]` seems like that may be overqualified for a selector? Do we need the anchors tags in the selector at all?